standard.tex

```diff
@@ -26,67 +26,76 @@
 \end{titlepage}
 % end titlepage -- }}}
 
-%start of guiding principles section -- {{{
+%Guiding Principles --  {{{
 \section{Guiding Principles of all Decisions}
 This document exists to set out a standard for all the electrical gear (avionics, telemetry, ground support equipment). Changes/revisions are very welcome. All design decisions on mission critical electrical systems should be made with the following four points in mind:
 
 \begin{enumerate}
-\item Our electrical stuff needs to work 100\% of the time at competition. If that's unattainable, 6 sigma is also acceptable. A lot of work from a lot of people go into building this rocket, and missing our 1 launch attempt because of an electrical problem is not acceptable.
+\item Our electrical stuff needs to work 100\% of the time at competition. A lot of work from a lot of people go into building this rocket, and missing our 1 launch attempt because of an electrical problem is not acceptable.
```

Comment:
Might be worth adding that this is true of engine tests too (to a lesser degree). Maybe institutionalize an electrical integration test at least 5 days prior to any engine test?

Reply:
I don't see that requirement as a guiding principle.